lib/platform/github/gh-got-wrapper.js

```diff
@@ -19,7 +33,13 @@ async function get(path, options, retries = 5) {
   opts.baseUrl = opts.endpoint;
   delete opts.endpoint;
   const method = opts.method || 'get';
-  const useCache = opts.useCache || true;
+  const useCache = opts.useCache || true; // in-memory cache
```

Comment:
This line is always true!

Reply:
so we always use cache
https://github.com/renovatebot/renovate/blob/45dba15b7464ee30fb76f2747a2909b49d3578f2/lib/platform/github/gh-got-wrapper.js#L27

Reply:
Whoops. perhaps should be `opts.useCache !== false`

Reply:
https://github.com/renovatebot/renovate/blob/master/lib/platform/bitbucket-server/bb-got-wrapper.js#L19-L20